advanced/Scripts/webpage.sh

```diff
@@ -456,6 +460,7 @@ main() {
 		"setdns"            ) SetDNSServers;;
 		"setexcludedomains" ) SetExcludeDomains;;
 		"setexcludeclients" ) SetExcludeClients;;
+		"halt"              ) Halt;;
```

Comment:
To me, "Halt" and "Reboot" are less ambiguous, whereas "shutdown" can imply either action.